thirdeye-spi/pom.xml

```diff
@@ -98,6 +98,14 @@
       <artifactId>checker-qual</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
following edits to this PR guice can be removed right?

Reply:
Yes, thanks for catching this
done